src/Foundation/NSMetadataItem.cs

```diff
@@ -162,1053 +169,1347 @@ bool GetBool (NSString key)
 			}
 		}
 
-		[iOS (7,0)][Mac (10,9)]
 		public NSError UbiquitousItemDownloadingError {
 			get {
 				return Runtime.GetNSObject<NSError> (GetHandle (NSMetadataQuery.UbiquitousItemDownloadingErrorKey));
 			}
 		}
 
-		[iOS (7,0)][Mac (10,9)]
 		public NSError UbiquitousItemUploadingError {
 			get {
 				return Runtime.GetNSObject<NSError> (GetHandle (NSMetadataQuery.UbiquitousItemUploadingErrorKey));
 			}
 		}
 
 		// XAMCORE_4_0 FIXME return nullable
+#if !NET
 		[iOS (8,0)][Mac (10,10)]
+#endif
 		public bool UbiquitousItemDownloadRequested {
 			get {
 				return GetBool (NSMetadataQuery.UbiquitousItemDownloadRequestedKey);
 			}
 		}
 
 		// XAMCORE_4_0 FIXME return nullable
+#if !NET
 		[iOS (8,0)][Mac (10,10)]
+#endif
 		public bool UbiquitousItemIsExternalDocument {
 			get {
 				return GetBool (NSMetadataQuery.UbiquitousItemIsExternalDocumentKey);
 			}
 		}
 
+#if !NET
 		[iOS (8,0)][Mac (10,9)]
```

Comment:
Mac is already included in the class. Is it worth removing?

Reply:
yes